tests/test_cookiecutter_generation.py

```diff
@@ -73,9 +73,22 @@ def context():
     {"keep_local_envs_in_vcs": "n"},
     {"debug": "y"},
     {"debug": "n"},
+    {"mail_service", "AWS SES"},
+    {"mail_service", "Mailgun"},
+    {"mail_service", "Mailjet"},
+    {"mail_service", "Mandrill"},
+    {"mail_service", "Postmark"},
+    {"mail_service", "Sendgrid"},
+    {"mail_service", "SendinBlue"},
+    {"mail_service", "SparkPost"},
+    {"mail_service", "Other SMTP"},
```

Comment:
```suggestion
    {"mail_service": "AWS SES"},
    {"mail_service": "Mailgun"},
    {"mail_service": "Mailjet"},
    {"mail_service": "Mandrill"},
    {"mail_service": "Postmark"},
    {"mail_service": "Sendgrid"},
    {"mail_service": "SendinBlue"},
    {"mail_service": "SparkPost"},
    {"mail_service": "Other SMTP"},
```

Reply:
Yea, running too tests right now. @browniebroke UNSUPPORTED_COMBINATIONS running into errors with the second one.

Reply:
I'm having trouble with one test:
```
UNSUPPORTED_COMBINATIONS = [
    {"cloud_provider": "GCP", "mail_service": "Amazon SES"},
    {"cloud_provider": "None", "use_whitenoise": "n"},
    {"cloud_provider": "None", "mail_service": "Amazon SES"},
]
```
Error:
```

=================================================== FAILURES ====================================================
_________________________________ test_error_if_incompatible[invalid_context1] __________________________________
tests/test_cookiecutter_generation.py:208: in test_error_if_incompatible
    assert result.exit_code != 0
E   assert 0 != 0
E    +  where 0 = <Result /private/var/folders/_x/y10g5kjj0yxbklqsmxmr_3qr0000gq/T/pytest-of-andrewwork/pytest-4/test_error_if_incompatible_inv1/cookies/bake00/my_test_project>.exit_code
```
Coming from this test:
```
@pytest.mark.parametrize("invalid_context", UNSUPPORTED_COMBINATIONS)
def test_error_if_incompatible(cookies, context, invalid_context):
    """It should not generate project an incompatible combination is selected."""
    context.update(invalid_context)
    result = cookies.bake(extra_context=context)

    assert result.exit_code != 0
    assert isinstance(result.exception, FailedHookException)
```